lib/install/management/appliance.go

```diff
@@ -581,6 +586,58 @@ func (d *Dispatcher) waitForKey(key string) {
 	return
 }
 
+// isPortLayerRunning decodes the `docker info` response to check if the portlayer is running
+func isPortLayerRunning(res *http.Response) bool {
+	defer res.Body.Close()
+	resBody, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		return false
+	}
+
+	var sysInfo dockertypes.Info
+	if err = json.Unmarshal(resBody, &sysInfo); err != nil {
+		return false
+	}
+
+	for _, status := range sysInfo.SystemStatus {
+		if status[0] == sysInfo.Driver {
+			return status[1] == "RUNNING"
+		}
+	}
+
+	return false
+}
+
+// waitForApplianceInit checks if the appliance is live by issuing `docker info` to it
+func waitForApplianceInit(hostIP, port string) bool {
+	dockerInfoURL := "https://" + hostIP + ":" + port + "/info"
+	tr := &http.Transport{
+		TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
```

Comment:
Isn't there a cert that matches the docker endpoint available in the `VirtualContainerHostConfigSpec`?